second-analysis-steps/advanced-dirac.md

```diff
@@ -0,0 +1,209 @@
+# Advanced Dirac
+
+At this point we have used Ganga for a number of things related to DaVinci and basic file management. However, Ganga is a much more flexible tool than that. Using `Python` and `IPython` we can set up more complicated workflows that manage much of your analysis for you!
+
+{% objectives "Learning Objectives" %}
+
+* Learn core concepts of the GangaTasks package
```

Comment:
In these bullet points you don't use full stops and in the ones lower down you do. I'll add them in here to be consistent with other lessons